llvm/lib/Analysis/CodeMetrics.cpp

```diff
@@ -111,11 +112,22 @@ void CodeMetrics::collectEphemeralValues(
   completeEphemeralValues(Visited, Worklist, EphValues);
 }
 
+static bool isUsedOutsideOfLoop(const Instruction &I, const Loop &L) {
+  for (const auto *U : I.users()) {
+    if (auto *I = dyn_cast<Instruction>(U)) {
```

Comment:
All users of Instructions are Instructions; just `cast<>`.

Reply:
Fixed.